packages/compass/release/publish.js

```diff
@@ -62,8 +38,7 @@ async function waitGithubRelease(releaseVersion, { github, changelog }) {
   cli.action.start(`Checking if Github release ${chalk.bold(releaseVersion)} really is published.`);
   const alreadyPublished = await github.isReleasePublished(releaseVersion);
   if (!alreadyPublished) {
-    cli.error('Release is not published yet - did you really publish?');
-    cli.info('Please publish the release on Github and run publish again to verify it worked.');
+    cli.error('Release is not published yet - did you really publish? Please publish the release on Github and run publish again to verify it worked.');
```

Comment:
I would just remove `npm run release publish`, at this point everything from changelog to publishing the manifest is outdated, we don't really need it anymore.

Means we can ditch also the changelog code and many more

Reply:
I can do the cleanup in a follow-up PR if you want, we should also get rid of the checks that still require AWS keys locally

Reply:
Oh, I actually thought that the changelog for the releases is still something that you need to manually apply in GitHub. is it not true anymore? How does the changelog message gets set now?

I can do a more through clean up in the same PR, thanks for highlighting! I just genuinely thought that this is still required 😅

Reply:
Ahhh, it's this GitHub releases thingie, now I remember 🤦